src/scss/component/accordion/_index.scss

```diff
@@ -0,0 +1,111 @@
+// @gruapl/graupl accordion component styles.
+
+@use "defaults";
+@use "variables" as *;
+@use "../../defaults" as root-defaults;
+@use "../../mixins/layer" as *;
+
+#{defaults.$accordion-selector} {
+  @include layer(component) {
+    border-radius: #{$accordion-border-radius};
+    border: #{$accordion-border};
+
+    #{defaults.$accordion-item-selector} {
+      &>*:first-child {
+        margin: 0;
+
+        #{defaults.$accordion-trigger-selector} {
+          border-style: none;
+        }
+      }
+    }
+
+    #{defaults.$accordion-trigger-selector} {
+      background: #{$accordion-trigger-background};
+      color: #{$accordion-trigger-text-color};
+      display: block;
+      font-size: #{$accordion-trigger-font-size};
+      font-weight: #{$accordion-trigger-font-weight};
+      margin: 0;
+      padding: #{$accordion-trigger-padding};
+      position: relative;
+      text-align: #{$accordion-trigger-text-align};
+      width: 100%;
+      outline: none;
+
+      &:hover,
+      &:focus {
+        background-color: #{$accordion-trigger-background-active};
+      }
+
+      &:focus {
+        outline: 4px solid transparent;
+      }
+
+      &:focus #{defaults.$accordion-icon-selector},
+      &:hover #{defaults.$accordion-icon-selector} {
+        border-color: hsl(216deg 94% 43%);
+      }
+
+      &[aria-expanded="true"] #{defaults.$accordion-icon-selector} {
+        transform: translateY(-50%) rotate(-135deg);
+      }
+
+      &:focus #{defaults.$accordion-title-selector} {
+        border-color: hsl(216deg 94% 43%);
+      }
+    }
+
+    #{defaults.$accordion-title-selector} {
+      display: block;
+      pointer-events: none;
+      border: #{$accordion-trigger-title-border};
+      border-radius: #{$accordion-trigger-title-border-radius};
+      padding: #{$accordion-trigger-title-padding};
+      outline: none;
+    }
+
+    #{defaults.$accordion-icon-selector} {
+      border: solid currentcolor;
+      border-width: 0 2px 2px 0;
+      height: 0.5rem;
+      pointer-events: none;
+      position: absolute;
+      right: 2em;
+      top: 50%;
+      transform: translateY(-60%) rotate(45deg);
+      width: 0.5rem;
+    }
+
+    #{defaults.$accordion-panel-selector} {
+      margin: 0;
+      padding: #{$accordion-panel-padding};
+      background-color: #{$accordion-panel-padding};
+    }
+
+    .hide {
+      #{defaults.$accordion-panel-selector} {
+        // display: none;
+        overflow: hidden;
+        max-height: 0;
+        padding: 0;
+      }
+    }
+
+    // TODO: See how I can use the transitioning class to transiton open the accordion.
```

Comment:
What aspect of this do you need help on? Like which transitions to use? How to handle animations on/off? How to get it to properly work with the JavaScript?

Reply:
This is working now.